src/Core/src/Handlers/ScrollView/ScrollViewHandler.Android.cs

```diff
@@ -24,6 +24,7 @@ protected override void ConnectHandler(MauiScrollView platformView)
 		{
 			base.ConnectHandler(platformView);
 			platformView.ScrollChange += ScrollChange;
+			platformView.CrossPlatformArrange = VirtualView.CrossPlatformArrange;
```

Comment:
Are we missing a null set on the detach? How do we prevent memory leaks with this?

Or can't this happen in this case?

Reply:
Just missed it. Fixed.